sensirion_i2c_hal.h

```diff
@@ -56,6 +56,11 @@ int16_t sensirion_i2c_hal_select_bus(uint8_t bus_idx);
  */
 void sensirion_i2c_hal_init(void);
 
+/**
+ * Initialize driver by setting device initialized in 3rd party
+ */
+void sensirion_i2c_hal_set_dev(void *dev);
```

Comment:
This API is specific for a platform, so shouldn't be present in the HAL. I would rather want to see it in a separate header file in the `zephyr_user_space/` directory.